README.md

```diff
@@ -16,14 +16,16 @@ Implements the `hook_permission` hook to add the `"admin.analytics.view"` permis
 
 ### hook_analytics_widget
 
-Return value:
-`id`: identifier, optional. It can be used to remove or modify the widget using the alter hook.  
+Return value is an array of objects in the following format for each widget object:  
+
+`id`: identifier, optional. It can be used to find and remove or modify the widget using the alter hook.  
 `content`: HTML string, the content of the admin widget  
-`weight`: number, optional: the order of the widget on the admin page
+`weight`: number, optional: the order of the widget on the admin page  
+`permission`: string, optional: the permission needed to view the widget. Default value: `"admin.analytics.view"`
```

Comment:
I think it would be nice to set an optional `access_callback` function because `permission` module can handle it.